.github/scripts/deploy_to_pypi.sh

```diff
@@ -1,4 +1,9 @@
 #!/bin/sh
 
+action=$1
 # Upload package to PyPI using github secrets
-twine upload -u __token__ -p "$TWINE_PASSWORD" dist/*
\ No newline at end of file
+if [[ $action == "live_deploy" ]]; then
+    twine upload -u __token__ -p "$PYPI_KEY" dist/*
+else
+    twine upload -u __token__ -p "$TEST_PYPI_KEY" --repository testpypi dist/*
```

Comment:
We can probably consolidate `PYPI_KEY` and `TEST_PYPI_KEY` into a single env var that is passed in the workflows. This way, we don't pass a valid production key for pypi when running test deploys.